mautic/mautic
# [UI] Contact details

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependencies:
- #13940
- #13929
- #13919

This PR reorganizes the Contact details page.




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/55669d93-6439-4694-8a31-292830e2721f)  | ![image](https://github.com/user-attachments/assets/e3163c2b-8d49-464c-a8b3-57c05669e561)

_No changes were made to font-size, I'm just using a big screen with high resolution._

![image](https://github.com/user-attachments/assets/84a0a01c-e13e-42fc-a44c-37229a2e067d)

![image](https://github.com/user-attachments/assets/a87a4118-a416-4d06-b618-47393db1751c)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create a contact
4. Save & Close
5. See their details
